themes/bootstrap3/js/lightbox.js

```diff
@@ -278,7 +278,10 @@ VuFind.register('lightbox', function Lightbox() {
       data: data
     }).done(function recaptchaReset() {
       if (typeof grecaptcha !== 'undefined') {
-        grecaptcha.reset($(form).find('.g-recaptcha').data('captchaId'));
+        var captcha = $(form).find('.g-recaptcha');
```

Comment:
This captcha-related change appears to pop up several times in this PR. Is this related to the form change, or is this an unrelated improvement? Do we perhaps want a `resetReCaptcha(form)` function in common.js so we don't have to repeat the same four lines of code multiple times, or is that overkill?